src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,154 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    public static final int MIN_AGE = 18;
+    private static final String CORRECT_LOGIN = "newUser";
+    private static final String CORRECT_PASSWORD = "123456";
+    private static final String TOO_SHORT_PASSWORD = "12345";
+    private static RegistrationService regService;
+    private User correctUser;
+
+    @BeforeAll
+    static void init() {
+        regService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        correctUser = createCorrectUser();
+    }
+
+    @AfterEach
+    void cleanUp() {
+        Storage.people.clear();
+    }
+
+    private User createCorrectUser() {
+        User user = new User();
+        user.setAge(MIN_AGE);
+        user.setLogin(CORRECT_LOGIN);
+        user.setPassword(CORRECT_PASSWORD);
+        return user;
+    }
+
+    @Test
+    void register_UserNull_NotOk() {

```

Comment:
Let's follow our naming convention and start work with lowercase letter after '_':
```suggestion
    void register_nullUser_notOk() {

```